changes/2020-07-13_detect-base64-encoded-messages/change.md

```diff
@@ -0,0 +1,88 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Detect Base64-encoded Messages
+
+## Affected Features
+
+This serves as a reference of all features that this change affects.
+
+| Feature                                                                                                                                     |
+| ------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Decrypt](https://github.com/awslabs/aws-encryption-sdk-specification/blob/623992d999db0b309d8a8adbd664f0d72feee813/client-apis/decrypt.md) |
+|                                                                                                                                             |
+
+## Affected Specifications
+
+This serves as a reference of all specification documents that this change affects.
+
+| Specification                           |
+| --------------------------------------- |
+| [Decrypt](../../client-apis/decrypt.md) |
```

Comment:
Should the SHA be here?

Reply:
I was following the example here, which only includes the SHA in "Affected Features" but not in "Affected Specifications": https://raw.githubusercontent.com/awslabs/aws-encryption-sdk-specification/master/changes/2020-05-13_remove-keyring-trace/change.md

@mattsb42-aws Do you think we need to use permalinks in both places? Whatever we decide should be written down in #137.

Reply:
As per my comment on #137, switching the existing permalink back to relative.